### PR TITLE
ref(starfish): Move extra endpoint details to new Endpoint Overview page

### PR DESCRIPTION
**Pull request by 0Calories**
The endpoint side panel was getting cluttered with way too much information, so we're moving some of the data out into its own page.

When you click on the sidepanel, there is a button that will take you to the endpoint overview page (design TBD). For now, on this page, there is mostly a lot of copypasta from the sidepanel, but eventually we want to surface more detailed and useful information like tag populations correlated with an increase in p50.

Note that this page has a lot of copy pasta that will need to be cleaned up eventually if we stick with it

image

### CODE REVIEW

**Comment by 0Calories** [comment]
The query for the DB table appears to be broken since converting the query to use the new wrapper hook, going to investigate further and attempt to fix before merging.

**Comment by 0Calories** [comment]
Fixed

**Review by edwardgou-sentry — APPROVED** [review]

File: static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  const theme = useTheme();
+
+  const {endpoint: transaction, method, statsPeriod} = location.query;
+  const pageFilter = usePageFilters();
+
+  const {
+    isLoading: isTableDataLoading,
+    data: tableData,
+    isRefetching: isTableRefetching,
+  } = useQueryMainTable({action: 'ALL', table: 'ALL'});
```
Comment (DominikB2014) [inline static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx:101]:
```suggestion
  } = useQueryMainTable({});
```
Small nit, we now treat action = 'ALL' and action = undefined as the same thing, same with table. so u can omit these params.

**Review by DominikB2014 — APPROVED** [review]

File: static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx (hunk 1 above)
Comment (0Calories) [reply static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx:101]: Ah nice, even better!

**Pull request merged**